src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/slider/SliderBuilderComponentObject.java

```diff
@@ -25,10 +25,6 @@ public SliderBuilderComponentObject(WebDriver driver) {
     // TODO Auto-generated constructor stub
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1135)